docs/content/latest/drivers-orms/python/_index.md

```diff
@@ -0,0 +1,58 @@
+menu:
+  latest:
+    identifier: python-drivers
+    parent: drivers-orms
+    weight: 570
+isTocNested: true
+showAsideToc: true
+---
+
+Following are the recommended projects for implementing Python Applications for YugabyteDB YSQL API.
+
+| Project | Type | Support Level |
+| :------ | :--- | :------------ |
+| [Yugabyte Psycopg2](yugabyte-psycopg2) (Recommended) | Python Driver | Full |
+| [Postgres Psycopg2](postgres-psycopg2) | Python Driver | Full |
+| [SQLAlchemy](sqlalchemy) | ORM |  Full |
+| [Django](django) | ORM |  Full |
+
+## Build a Hello World App
+
+Learn how to establish a connection to YugabyteDB database and begin simple CRUD operations using the steps in [Build an Application](/latest/quick-start/build-apps/python/ysql-psycopg2) in the Quick Start section.
+
+## Pre-requisites for Building a Python Application
+
+### Install Python
+
+Make sure that your system has Python3 installed. To check the version of Python installed, use the following command.
+```
+$ python -V
+```
+If not already installed, download and install it from [Python's Website](https://www.python.org/downloads/).
+
+### Create a Python Project
+
+Create a python file by adding ```.py``` extension to your filename. It is also advised to start a virtual environment to keep dependencies required by different projects separate. Make sure pip is also installed in the environment.
+
+### Create a YugabyteDB Cluster
+
+Set up a Free tier Cluster on [Yugabyte Anywhere](https://www.yugabyte.com/cloud/). The free cluster provides a fully functioning YugabyteDB cluster deployed to the cloud region of your choice. The cluster is free forever and includes enough resources to explore the core features available for developing the Java Applications with YugabyteDB database. Complete the steps for [creating a free tier cluster](latest/yugabyte-cloud/cloud-quickstart/qs-add/).
+
+Alternatively, You can also setup a standalone YugabyteDB cluster by following the [install YugabyteDB Steps](/latest/quick-start/install/macos).
+
+## Usage Examples
+
+For fully runnable code snippets and exaplantion for common operations, see the specific Java driver and ORM section. Below table provides quick links for navigating to driver specific documentation and also the usage examples.
+
+| Project | Type | Usage Examples |
+| :------ | :--- | :------------- |
+| [Yugabyte Psycopg2](/latest/reference/drivers/python/yugabyte-psycopg2-reference/) | Python Driver | [Hello World]() <br />[CRUD App](yugabyte-psycopg2)
+| [Postgres Psycopg2](/latest/reference/drivers/python/postgres-psycopg2-reference/) | Python Driver | [Hello World](/latest/quick-start/build-apps/python/ysql-psycopg2) <br />[CRUD App](postgres-psycopg2)|
+| [SQLAlchemy](sqlalchemy) | ORM |  [Hello World](sqlalchemy) <br />[CRUD App](/latest/quick-start/build-apps/python/ysql-sqlalchemy) |
+| [Django](django) | ORM | [Hello World](django) <br />[CRUD App](/latest/quick-start/build-apps/python/ysql-django) |
```

Comment:
SQLAlchemy and Django - hello world links to driver, while crud links to hello world

Reply:
fixed.